web/.eslintrc.json

```diff
@@ -20,6 +20,25 @@
     "prettier"
   ],
   "rules": {
+    "import/order": [
+      "error",
+      {
+        "groups": ["external", "internal"],
```

Comment:
If we stick with `import/order` (see other comment), I think we should use the full grouping they suggest:
`["builtin", "external", "internal", "parent", "sibling", "index", "object", "type"]`